packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -338,14 +282,16 @@ export default connect(
     profile: profile(state),
     plans: plans(state),
     createSubscriptionStatus: createSubscriptionStatus(state),
+    updateSubscriptionPlanStatus: updateSubscriptionPlanStatus(state),
     plansByProductId: plansByProductId(state),
   }),
   {
-    resetCreateSubscription: resetCreateSubscription,
-    resetCreateSubscriptionError: resetCreateSubscription,
+    resetCreateSubscription,
     fetchProductRouteResources,
-    createSubscription: createSubscriptionAndRefresh,
```

Comment:
I don't know why I renamed `createSubscriptionAndRefresh` as `createSubscription` here, but I decided using the longer name might be more clear.